jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codemodernizer/CodeWhispererCodeModernizerSessionTest.kt

```diff
@@ -236,6 +236,45 @@ class CodeWhispererCodeModernizerSessionTest : CodeWhispererCodeModernizerTestBa
         }
     }
 
+    @Test
+    fun `CodeModernizerSession can create zip and replace Windows file path`() {
+        addFilesToProjectModule(
+            "src\\tmp.java",
+            "target\\smth.java",
+            "target\\somedir\\anotherthing.class",
+            "pom.xml",
+            "someModule\\pom.xml",
+            "someModule\\target\\smth.class",
+            "someModule\\src\\helloworld.java",
+        )
+        var file: File? = null
+        val rootManager = ModuleRootManager.getInstance(module)
+        val roots = rootManager.contentRoots
+        assertFalse(roots.isEmpty() || roots.size > 1)
+
+        val pom = roots[0].children.first { it.name == "pom.xml" }
+        val context = CodeModernizerSessionContext(project, pom, JavaSdkVersion.JDK_1_8, JavaSdkVersion.JDK_11)
+        runInEdtAndWait {
+            file = context.createZipWithModuleFiles().payload
+        }
+        assertNotNull(file)
+        val zipFile = ZipFile(file)
+        val entries = zipFile.entries()
+        while (entries.hasMoreElements()) {
+            val entry = entries.nextElement() ?: continue
+            val fileContent = zipFile.getInputStream(entry).bufferedReader().readLine()
```

Comment:
shouldn't this be `readText()` instead?

Reply:
Do you mean using the Kotlin version of Reader.readText()? What benefit will it introduce over using the JavaIO BufferReader.readLine()? (Each file added to the test project fixture only contains a line.)